integration-tests/src/test/java/org/apache/druid/tests/security/ITTLSTest.java

```diff
@@ -396,8 +396,9 @@ private void checkFailedAccess(
       catch (RuntimeException re) {
         Throwable rootCause = Throwables.getRootCause(re);
 
-        if (rootCause instanceof IOException && "Broken pipe".equals(rootCause.getMessage())) {
-          if (retries > MAX_BROKEN_PIPE_RETRIES) {
+        if (rootCause instanceof IOException && ("Broken pipe".equals(rootCause.getMessage())
+                                                 || "Connection reset by peer".contains(rootCause.getMessage()))) {
+          if (retries > MAX_CONNECTION_EXCEPTION_RETRIES) {
             Assert.fail(StringUtils.format(
                 "Broken pipe retries exhausted, test failed, did not get %s.",
```

Comment:
Broken pipe or connection reset retries exhausted.

Reply:
Changed the error message.